### ISSUE TITLE
Retina Support on Mac Request

### ISSUE THREAD

**Issue by ZhaoZuohong** [mortbopet/Ripes#79]
Ripes does not support Retina display.

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/34163622/113636199-af9c6f80-96a4-11eb-974a-1d83d169b2a0.png">

**Comment by mortbopet** [comment]
Hi @ZhaoZuohong 
Sorry for the delay. Is the issue that the icons in the toolbar are rendered too large?

**Comment by ZhaoZuohong** [comment]
No, the size of the icons are correct, but the whole interface is fuzzy, and ppi is low.

**Comment by mortbopet** [comment]
Marking this as _help wanted_ since I don't have access to a retina display, so it's a bit tricky for me to work on/verify a solution.
